Doc/library/sqlite3.rst

```diff
@@ -1041,19 +1041,34 @@ Connection objects
          (2, 'broccoli pie', 'broccoli cheese onions flour')
          (3, 'pumpkin pie', 'pumpkin sugar flour butter')
 
-   .. method:: load_extension(path, /)
+   .. method:: load_extension(path, /, *, entrypoint=None)
 
-      Load an SQLite extension from a shared library located at *path*.
+      Load an SQLite extension from a shared library.
       Enable extension loading with :meth:`enable_load_extension` before
       calling this method.
 
+      :param str path:
+
+         The path to the SQLite extension.
+
+      :param entrypoint:
+
+         Optional entry point name.
+         If ``None`` (default),
```

Comment:
I feel like "optional" is already clear from the signature given above (but don't feel strongly about it)

```suggestion
         Entry point name.
         If ``None`` (the default),
```